src/main/resources/db/changelog/init/15062021_create_user_discount_table.yml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1234567890-7
+      author: tesliuk.ua@gmail.com
+      changes:
+        - createTable:
+            schemaName: public
+            tableName: user_discount
+            columns:
+              - column:
+                  name: usr_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ud_usr_id
+                    foreignKeyName: fk_ud_user
+                    references: user(usr_id)
+              - column:
+                  name: dis_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ud_dis_id
```

Comment:
rename to pk_user_discount_id (should be the same as previous)

Reply:
Changed in other PR